home/jobs/OMERO-web/config.xml

```diff
@@ -187,8 +201,8 @@ omero config set omero.web.public.password public
 omero config set omero.web.public.url_filter &apos;^/&apos;
 
 # QA feedback configuration
-omero config set omero.web.feedback.comment.enabled false
-omero config set omero.web.feedback.error.enabled false
+# omero config set omero.web.feedback.comment.enabled false
+# omero config set omero.web.feedback.error.enabled false
```

Comment:
```suggestion
```
Presumably this was a temporary change for testing https://github.com/ome/openmicroscopy/pull/5957 that was never properly reverted?

Reply:
So you'd be in favor of re-enabling in merge-ci and latest-ci ?

Reply:
Definitely